lib/assessment.js

```diff
@@ -32,14 +38,24 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
-
+  var newArray = [];
+  array.forEach(function(c, i, a){
+    if(callback(c))
+      newArray.push(c);
+  })
+  return newArray;
 };
 
 // Question 5
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
-
+  var result = [];
+  array.forEach(function(c, i, a){
+    if(callback(c, i, a))
+      result.push(c);
+  })
+  return result[0];
```

Comment:
Sure, this will work if there's at least one match. However, if there are no matches, it may act a little funky.The actual `find` will return `-1` if something can't be found. Can you see if there might be a way to do that?